src/commands/cmd_stream.cc

```diff
@@ -258,6 +258,30 @@ class CommandXGroup : public Commander {
       return Status::OK();
     }
 
+    if (subcommand_ == "setid") {
+      if (args.size() != 5 && args.size() != 7) {
+        return {Status::RedisParseErr, errWrongNumOfArguments};
+      }
+
+      xgroup_create_options_.last_id = GET_OR_RET(parser.TakeStr());
```

Comment:
Should we check here if the received ID is in a proper format? I mean int_value-int_value/int_value/etc.

Reply:
It is checked at this [line](https://github.com/apache/kvrocks/pull/1833/files#diff-127fef0065aa75a681fdfe32519927b9c2a5724ce2de8bc5e19cdded49ac6b2eR418)

Reply:
I see, thank you.